website/docs/r/compute_backend_service.html.markdown

```diff
@@ -17,31 +17,31 @@ For internal load balancing, use a [google_compute_region_backend_service](/docs
 ## Example Usage
 
 ```hcl
-resource "google_compute_backend_service" "foobar" {
-  name        = "blablah"
-  description = "Hello World 1234"
+resource "google_compute_backend_service" "website" {
```

Comment:
These doc changes are awesome! Looks way better than names like `"blablah"` and have `test` appear in the name of most resources.